packages/cli-kit/src/private/node/ui/components/TextPrompt.tsx

```diff
@@ -53,14 +62,15 @@ const TextPrompt: FunctionComponent<TextPromptProps> = ({
     [allowEmpty, validate],
   )
 
-  const {oneThird} = useLayout()
+  const {oneThird, twoThirds} = useLayout()
   const [answer, setAnswer] = useState<string>('')
   const answerOrDefault = answer.length > 0 ? answer : defaultValue
   const displayEmptyValue = answerOrDefault === ''
   const displayedAnswer = displayEmptyValue ? emptyDisplayedValue : answerOrDefault
   const {exit: unmountInk} = useApp()
   const [submitted, setSubmitted] = useState(false)
-  const [error, setError] = useState<string | undefined>(undefined)
+  const [cancelled, setCancelled] = useState(false)
```

Comment:
I think it would be better to introduce an enum for the state like in AutocompletePrompt. I'm also planning to refactor the state into a separate hook.